apps/widget-configurator/src/app/configurator/controls/CurrentTradeTypeControl.tsx

```diff
@@ -9,19 +9,20 @@ import Select from '@mui/material/Select'
 
 import { TRADE_MODES } from '../consts'
 
+const label = 'Current trade type'
```

Comment:
Can you make these conts CAPITAL to not confuse with local variables?

Reply:
Sure! Fixed